drivers/gpu/drm/panel/panel-samsung-s6e3fa0-full.c

```diff
@@ -0,0 +1,764 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2022, Adam Honse <calcprogrammer1@gmail.com>
+// Copyright (c) 2021, Alexey Minnekhanov <alexeymin@postmarketos.org>
+// Copyright (c) 2021, The Linux Foundation. All rights reserved.
+
+#include <linux/backlight.h>
+#include <linux/delay.h>
+#include <linux/gpio/consumer.h>
+#include <linux/module.h>
+#include <linux/of.h>
+#include <linux/regulator/consumer.h>
+
+#include <drm/drm_mipi_dsi.h>
+#include <drm/drm_modes.h>
+#include <drm/drm_panel.h>
+
+#include <video/mipi_display.h>
+
+/* Panel variants */
+#define LCD_ID_S6E3FA0		0x414024
```

Comment:
For single panel driver you don't need IDs and the whole s6e3fa0_detect_panel_type() func

Reply:
Left in panel ID check but cleaned up some unused code